# La stima del punteggio vero {#sec-err-std-stima}

::: {.chapterintro data-latex=""}
Nella Teoria Classica dei Test, la stima del punteggio vero con il metodo della regressione di Kelley è una correzione del punteggio osservato, basata sull'affidabilità del test, per ottenere una stima più accurata del punteggio "vero" di un individuo. La formula di Kelley riduce la componente di errore nel punteggio osservato attenuando le estremità, poiché punteggi molto alti o bassi tendono ad avvicinarsi alla media. La stima è calcolata come una media ponderata tra il punteggio osservato e la media dei punteggi, pesata in base all'affidabilità del test.
:::

**Prerequisiti**

**Concetti e Competenze Chiave**

**Preparazione del Notebook**

In [1]:
# Carica il file _common.R per impostazioni di pacchetti e opzioni
here::here("code", "_common.R") |> source()

# Carica pacchetti aggiuntivi
pacman::p_load(psychometric)

## Introduzione

Uno dei principali scopi della valutazione psicologica è stimare il punteggio vero del soggetto. Il punteggio osservato $X$ differisce dal punteggio vero $T$ a causa dell'errore di misurazione: $X = T + E$. Ora poniamoci l'obiettivo di utilizzare i concetti della Teoria Classica per stimare il punteggio vero di un soggetto, utilizzando il suo punteggio osservato e l'affidabilità del test. Questa stima risulta particolarmente utile quando è necessario costruire un intervallo di confidenza per il punteggio vero del soggetto.

Per costruire l'intervallo di confidenza del vero punteggio, sono necessarie due misurazioni:

- Una stima del vero punteggio.
- L'errore standard della stima (ossia, una stima della deviazione standard della distribuzione delle stime del punteggio vero che si otterrebbero se il test venisse somministrato infinite volte nelle stesse condizioni).

Cominciamo affrontando il problema della stima del vero punteggio.

## Il paradosso di Kelley

Negli anni '20, Kelly ha dimostrato come sia possibile stimare il punteggio vero del rispondente utilizzando un modello di regressione. La formula di Kelley si fonda sull'equivalenza algebrica che lega l'attendibilità al quadrato del coefficiente di correlazione tra i punteggi osservati e quelli veri. Pertanto, la stima del punteggio vero di un rispondente può essere calcolata nel seguente modo:

$$
\begin{equation}
\hat{T} = \mu_x + \rho  (X - \mu_x),
\end{equation}
$$ {#eq-kelly}

dove $X$ rappresenta il punteggio osservato, $\mu_x$ è la media dei punteggi ottenuti da tutti i partecipanti nel campione, e $\rho$ è l'attendibilità del test.

Quando l'attendibilità è perfetta ($\rho = 1$), il punteggio vero coincide con il punteggio osservato. Nel caso di attendibilità nulla (dove tutta la varianza è attribuibile all'errore di misurazione), la stima più accurata del punteggio vero è semplicemente la media del campione. Per valori di $\rho$ compresi tra 0 e 1, la stima del punteggio vero si discosta dal punteggio osservato in direzione della media campionaria. In questo modo, la stima del punteggio vero illustra il concetto di regressione verso la media dei punteggi osservati, considerando l'attendibilità del test.

La formula del punteggio vero può essere interpretata nel modo seguente: per stimare il vero punteggio di un individuo, si parte dalla media della distribuzione di tutti i partecipanti e si procede in direzione del punteggio osservato. Tuttavia, il punteggio osservato non viene raggiunto completamente; l'entità dello spostamento è proporzionale all'attendibilità del test. Ciò implica che la stima del punteggio vero di un individuo, a seconda del valore di $\rho$, tiene conto anche della sua posizione rispetto alla media del gruppo. Se il soggetto si trova al di sotto della media, la stima del punteggio vero sarà maggiorata e viceversa. Questo fenomeno è noto come il "paradosso di Kelley".

È cruciale mettere in evidenza una discrepanza tra la formula di Kelley e l'intuizione comune che suggerisce il punteggio osservato come una stima accurata del vero punteggio (cioè $\hat{T} = X$). Tuttavia, questo punto di vista è valido solo quando la misura è perfettamente attendibile ($\rho = 1$). Al contrario, quando $\rho = 0$, la formula di Kelley suggerisce di utilizzare la media dei punteggi osservati come stima del vero punteggio, implicando che il punteggio osservato non rifletta necessariamente il vero punteggio, ma solo l'errore di misurazione.

In pratica, è estremamente improbabile che $\rho$ sia esattamente uguale a zero. Invece, con valori di $\rho$ compresi tra 0 e 1, la stima del punteggio vero si troverà in una posizione intermedia tra il punteggio osservato e la media della popolazione. Per una comprensione più dettagliata di questo concetto, possiamo fare riferimento a Kelley (1947), il quale ha osservato che:

> This is an interesting equation in that it expresses the estimate of true ability as the weighted sum of two separate estimates, -- one based upon the individual's observed score, $X_1$ ($X$ nella notazione corrente) and the other based upon the mean of the group to which he belongs, $M_1$ ($\mu_x$ nella notazione corrente). If the test is highly reliable, much weight is given to the test score and little to the group mean, and vice versa.

Per chiarire l'@eq-kelly e la sua derivazione in termini di predizione del punteggio vero a partire dal punteggio osservato, iniziamo esaminando il modello di base di regressione lineare semplice. Questo modello stabilisce una relazione diretta tra il punteggio osservato $X$ e il punteggio vero $T$, una relazione che inizialmente abbiamo descritto con la formula $X = 0 + 1 \cdot T + E$. Il nostro interesse specifico qui, però, si sposta verso la predizione del punteggio vero $T$ utilizzando il punteggio osservato $X$, attraverso un modello di regressione. La formula per questa predizione assume la forma:

$$
T = \alpha + \beta X + \varepsilon.
$$

Riorganizzando le variabili in termini di deviazioni dalla loro media ($x = X - \bar{X}$ e $\tau = T - \mathbb{E}(T)$), e considerando l'intercetta $\alpha$ come 0, il modello si semplifica in $\hat{\tau} = \beta x$, dove $\hat{\tau}$ rappresenta la nostra stima del punteggio vero come deviazione dalla media. La questione centrale diventa quindi il calcolo di $\beta$, che è la pendenza della retta di regressione nel nostro modello semplificato.

Il valore di $\beta$ viene definito come $\beta = \frac{\sigma_{\tau x}}{\sigma^2_x}$, che ci permette di esprimere il modello come:

$$
\hat{\tau} = \frac{\sigma_{\tau x}}{\sigma^2_x} x.
$$

Introducendo la correlazione tra $x$ (o $X$) e $\tau$ (o $T$), denotata $\rho_{\tau x}$, e sostituendo la covarianza con il prodotto della correlazione per le deviazioni standard dei punteggi, riscriviamo l'equazione come:

$$
\hat{\tau} = \rho_{\tau x}\frac{\sigma_{\tau}}{\sigma_x} x.
$$

Questo ci porta a considerare la definizione di attendibilità, secondo cui la varianza del punteggio vero può essere espressa come $\sigma^2_{\tau} = \sigma^2_x \rho_{xx^\prime}$. La stima del punteggio vero, in termini di deviazioni dalla media, diventa quindi una funzione del coefficiente di attendibilità e della deviazione del punteggio osservato dalla sua media:

$$
\hat{\tau} = \rho_{\tau x} \sqrt{\rho_{xx^\prime}} x.
$$

Avendo dimostrato che $\rho^2_{\tau x} = \rho_{xx^\prime}$, possiamo ulteriormente semplificare la nostra stima del punteggio vero come:

$$
\hat{\tau} = \rho_{xx^\prime} x.
$$

Questo ci indica che la stima del punteggio vero (in termini di deviazioni dalla media) si ottiene moltiplicando il punteggio osservato, anch'esso espresso come deviazione dalla media, per il coefficiente di attendibilità.

Per ritornare alla formula in termini di punteggi grezzi, aggiungiamo la media dei punteggi osservati $\bar{X}$ alla nostra equazione, ottenendo così la stima del punteggio vero grezzo $\hat{T}$:

$$
\hat{T} = \rho_{XX^\prime} (X - \bar{X}) + \bar{X}.
$$

Espandendo e riorganizzando l'equazione, arriviamo a una forma che chiarisce la relazione tra la media dei punteggi osservati, il coefficiente di attendibilità, e il punteggio osservato grezzo:

$$
\hat{T} = \bar{X} + \rho_{XX^\prime} (X - \bar{X}).
$$

Questa equazione finale dimostra come la stima del punteggio vero grezzo possa essere calcolata regolando il punteggio osservato per la media dei punteggi e il coefficiente di attendibilità. 

Nel contesto di dati campionari, dove il coefficiente di attendibilità popolazionale $\rho_{XX^\prime}$ viene sostituito con il suo corrispettivo campionario $r_{XX^\prime}$, la formula diventa:

$$
\hat{T} = \bar{X} + r_{XX^\prime} (X - \bar{X}),
$$

offrendo un metodo pratico per stimare il punteggio vero di un individuo a partire dal suo punteggio osservato, con l'aggiunta della media dei punteggi osservati e il coefficiente di attendibilità campionario.

**Esercizio.** Posto un coefficiente di attendibilità pari a 0.80 e una media del test pari a $\bar{X} = 100$, si trovi una stima del punteggio vero per un rispondente con un punteggio osservato uguale a $X$ = 115.

La stima del punteggio vero $\hat{T}$ è uguale a 

$$
\begin{equation}
\begin{aligned}
\hat{T} &= \bar{X} + r_{XX^\prime}  (X - \bar{X})\notag\\
&= 100 + 0.80 \cdot (115 - 100) = 112.
\end{aligned}
\end{equation}
$$

In alternativa, possiamo usare la funzione `Est.true` del pacchetto `psychometric`.

In [3]:
Est.true(115, 100, .8)

[1] 112

## L'Errore Standard della Stima 

Nell'ambito del modello di regressione di Kelley, uno strumento fondamentale per valutare la precisione delle stime del punteggio vero ottenute dai punteggi osservati è l'*errore standard della stima*. Questo indice quantifica quanto le nostre stime del punteggio vero possano variare se ripetessimo il test più volte sotto le stesse condizioni. Denotiamo l'errore standard della stima con $\sigma_{\hat{T}}$, dove $\hat{T}$ rappresenta la stima del valore vero.

L'errore standard della stima è cruciale per comprendere quanto sia affidabile la stima del punteggio vero. Un errore standard più piccolo indica una stima più precisa. Matematicamente, l'errore standard della stima si calcola come:

$$
\sigma_{\hat{T}} = \sigma_X \sqrt{\rho_{XX^\prime} (1 -\rho_{XX^\prime})},
$$ {#eq-std-err-estimate}

dove $\sigma_X$ è la deviazione standard dei punteggi osservati, e $\rho_{XX^\prime}$ rappresenta il coefficiente di correlazione tra i punteggi osservati e i punteggi veri. Questa formula assume una distribuzione normale dei punteggi e una relazione lineare tra i punteggi osservati e i punteggi veri.

Per dati campionari, utilizziamo una formula leggermente modificata per calcolare l'errore standard della stima:

$$
s_{\hat{T}} = s_X \sqrt{r_{XX^\prime} (1-r_{XX^\prime})},
$$

qui $s_X$ indica la deviazione standard campionaria, e $r_{XX^\prime}$ è il coefficiente di affidabilità campionario.

## Derivazione dell'Errore Standard della Stima

Per derivare l'@eq-std-err-estimate, iniziamo con la definizione dell'errore $\varepsilon$, che rappresenta la discrepanza tra il punteggio reale $T$ e il punteggio stimato $\hat{T}$, come illustrato nella formula:

$$
\varepsilon = T - \hat{T}.
$$

Si sottolinea la distinzione tra l'errore di misurazione, indicato con $E = X - T$ (dove $E$ quantifica l'errore di misurazione, ovvero la differenza tra il punteggio osservato $X$ e il punteggio reale $T$), e l'errore $\varepsilon = T - \hat{T}$ (che esprime la discrepanza tra il punteggio reale $T$ e la sua stima $\hat{T}$).

Adottando la formula $\hat{T} = \bar{X} + \rho_{XX^\prime} (X - \bar{X})$ per esprimere $\hat{T}$, si può calcolare la varianza di $\varepsilon$ come segue:

$$
\begin{aligned}
\mathbb{V}(\varepsilon) &=  \mathbb{V}(T - \hat{T}) \\
&= \mathbb{V}(T - \bar{X} - \rho_{XX^\prime} X + \rho_{XX^\prime}\bar{X}).
\end{aligned}
$$

Dato che l'aggiunta di una costante non altera la varianza di una variabile aleatoria, possiamo semplificare l'espressione a:

$$
\mathbb{V}(\varepsilon) = \mathbb{V}(T - \rho_{XX^\prime}X).
$$

Sfruttando la regola della varianza per la somma di variabili aleatorie, incluso il caso in cui una variabile è moltiplicata per una costante, arriviamo a:

$$
\begin{aligned}
\mathbb{V}(\varepsilon) &= \mathbb{V}(T) + \rho_{XX^\prime}^2 \mathbb{V}(X) - 2 \rho_{XX^\prime} \mbox{Cov}(X,T) \\
&= \sigma^2_T + \rho_{XX^\prime}^2 \sigma^2_X - 2 \rho_{XX^\prime} \sigma_{XT}.
\end{aligned}
$$

Con $\sigma_{XT} = \sigma^2_T$, possiamo ridurre ulteriormente l'espressione a:

$$
\sigma^2_{\varepsilon} = \sigma^2_T + \left(\frac{\sigma_T^2}{\sigma_X^2}\right)^2 \sigma^2_X - 2 \frac{\sigma_T^2}{\sigma_X^2} \sigma_{XT}.
$$

Semplificando, otteniamo:

$$
\begin{aligned}
\sigma^2_{\varepsilon} &= \sigma^2_T + \frac{\sigma_T^4}{\sigma_X^4} \sigma^2_X - 2 \frac{\sigma_T^2}{\sigma_X^2} \sigma_{XT} \\
&= \sigma^2_T \left(1 + \frac{\sigma_T^2}{\sigma_X^2} - 2 \frac{\sigma_{XT}}{\sigma_X^2}\right).
\end{aligned}
$$

Infine, con $\sigma_{XT} = \sigma^2_T$, semplifichiamo a:

$$
\begin{aligned}
\sigma^2_{\varepsilon} &= \sigma^2_T \left(1 - \frac{\sigma_{T}^2}{\sigma_X^2}\right).
\end{aligned}
$$

Questo ci porta alla formula dell'errore standard della stima $\sigma_{\varepsilon}$:

$$
\begin{aligned}
\sigma_{\varepsilon} &= \sigma_T \sqrt{1 - \frac{\sigma^2_T}{\sigma^2_X}} \\
&= \sigma_T \sqrt{\frac{\sigma^2_X - \sigma^2_T}{\sigma^2_X}} \\
&= \frac{\sigma_T}{\sigma_X} \sqrt{\sigma^2_X - \sigma^2_T}.
\end{aligned}
$$

Considerando che $\sigma^2_X = \sigma^2_T + \sigma^2_E$, l'errore standard di stima diventa:

$$
\begin{aligned}
\sigma_{\varepsilon} &= \frac{\sigma_T}{\sigma_X} \sqrt{\sigma^2_E } \\
&= \frac{\sigma_T}{\sigma_X} \sigma_E \\
&= \sqrt{\rho_{XX^\prime}} \sigma_E.
\end{aligned}
$$

Dato che l'errore standard di misurazione è definito come $\sigma_E = \sigma_X \sqrt{1 - \rho_{XX^\prime}}$, possiamo concludere che:

$$
\begin{aligned}
\sigma_{\varepsilon} &= \sqrt{\rho_{XX^\prime}} \sigma_E \\
&= \sqrt{\rho_{XX^\prime}} \sigma_X \sqrt{1-\rho_{XX^\prime}} \\
&= \sigma_X \sqrt{\rho_{XX^\prime} (1 - \rho_{XX^\prime})}.
\end{aligned}
$$

Quest'ultima espressione dimostra come l'errore standard della stima sia determinato dalla deviazione standard dei punteggi osservati, modulata dal coefficiente di correlazione tra i punteggi osservati e i punteggi veri.

## Intervallo di confidenza per il punteggio vero

Siamo ora finalmente nelle condizioni di potere calcolare l'intervallo di confidenza per il punteggio vero. Conoscendo l'errore standard della stima $\sigma_{\hat{T}}$,  l'intervallo di confidenza per il punteggio vero è dato da:

$$
\hat{T} \pm z  \sigma_{\hat{T}},
$$ 

laddove $\hat{T}$ è la stima del punteggio vero e $z$ è il quantile della normale standardizzata al livello di probabilità desiderato. Se il campione è piccolo (minore di
30) è opportuno usare $t$ anziché $z$.

### Interpretazione

Notiamo che l'intervallo $\hat{T} \pm z \sigma_{\hat{T}}$ è centrato sulla
stima puntuale del valore vero e la sua ampiezza dipende sia dal livello
di confidenza desiderato (rappresentato dal quantile $z_{\frac{\alpha}{2}}$),
sia dal grado di precisione dello stimatore, misurato dall'errore standard
della stima, $\sigma_{\hat{T}} = \sigma_X \sqrt{\rho_{XX^\prime} (1 -\rho_{XX^\prime})}$.
È importante notare che l'errore standard della stima diventa sempre più grande
man mano che diminuisce l'attendibilità $\rho_{XX^\prime}$ del test.

L'intervallo di confidenza indica quanto l'imprecisione della misura influisce sull'interpretazione dei dati. Più l'intervallo di confidenza è ampio, maggiore è l'incertezza nella valutazione dei risultati. 

**Esercizio.** @charter1996revisiting ha esaminato l'effetto della variazione dell'attendibilità del test sull'ampiezza dell'intervallo di confidenza per il punteggio vero. Utilizzando come esempio i punteggi di QI ($\mu$ = 100, $\sigma$ = 15), Charter ha immaginato di variare il coefficiente di attendibilità del test utilizzato per la misurazione del QI. I valori presi in considerazione sono 0.55, 0.65, 0.75, 0.85 e 0.95. Ad esempio, supponiamo di avere un punteggio osservato pari a QI = 120 e un coefficiente di attendibilità del test $\rho_{xx^\prime}$ pari a 0.65. In tali circostanze, la stima del punteggio vero è pari a

$$
\begin{equation}
\begin{aligned}
\hat{T} &= \bar{X} + r_{XX^\prime}  (X - \bar{X}) \notag\\
&= 100 + 0.65 (120 - 100)\notag\\
&= 113.\notag
\end{aligned}
\end{equation}
$$

L'errore standard della stima è uguale a

$$
\begin{equation}
\begin{aligned}
\sigma_{\hat{T}} &= \sigma_{X} \sqrt{r_{XX^\prime} (1 - r_{XX^\prime})} \notag\\
&= 15 \sqrt{0.65 (1 - 0.65)}\notag\\
&= 7.15.\notag
\end{aligned}
\end{equation}
$$

L'intervallo di confidenza al 95% per la stima del punteggio vero diventa pertanto uguale a 

$$
113 \pm 1.96 \cdot 7.15 = [98.98, 127.02].
$$ 

Si noti che si può calcolare l'errore standard della stima con la funzione `SE.Est()` del pacchetto `psychometric`.

In [4]:
SE.Est(15, .65)

[1] 7.154544

\
Inoltre, la funzione `CI.tscore()` restituisce sia la stima del punteggio vero sia l'intervallo di fiducia al livello desiderato di significatività. 

In [5]:
CI.tscore(120, 100, 15, 0.65, level = 0.95)

SE.Est   LCL      T.Score UCL     
1 7.154544 98.97735 113     127.0226

<!-- Lo stesso risultato si ottiene con la funzione `CI.tscore()` del -->
<!-- pacchetto `psychometric` che richiede i seguenti argomenti: il punteggio -->
<!-- osservato, la media del campione, la deviazione standard del campione e -->
<!-- l'attendibilità del test: -->

<!-- ```{r} -->
<!-- library("psychometric") -->
<!-- CI.tscore(120, 100, 15, 0.65) -->
<!-- ``` -->

## Cut-off

Gli intervalli di confidenza per il punteggio vero possono essere utilizzati per confrontare i limiti dell'intervallo con un cut-off. Ci sono tre possibili esiti: il limite inferiore dell'intervallo di confidenza è maggiore del cut-off, il limite superiore dell'intervallo è minore del cut-off, o il valore del cut-off è compreso all'interno dell'intervallo. Nel primo caso, lo psicologo può affermare, con un grado di certezza $1-\alpha$, che il valore vero del rispondente è superiore al cut-off. Nel secondo caso, lo psicologo può affermare, con un grado di certezza $1-\alpha$, che il valore vero del rispondente è inferiore al cut-off. Nel terzo caso, lo psicologo non può concludere né che il valore vero sia inferiore né che sia superiore al cut-off, con un certo grado di incertezza.

:::{.exercise}
Si considerino i punteggi del QI, per cui $\bar{X}$ = 100 e $s_X$ = 15.
Sia l'attendibilità del test $\rho_{XX^\prime}$ = 0.95. Supponiamo che il
rispondente abbia un QI = 130. Poniamo che il cut-off per ammettere il
rispondente ad un corso avanzato sia 120. Ci sono tre alternative: il
valore vero del rispondente è sicuramente maggiore di 120; il valore
vero del rispondente è sicuramente inferiore di 120; le evidenze
disponibili ci lasciano in dubbio se il punteggio vero sia maggiore o
minore di 120. Svolgiamo i calcoli per trovare l'intervallo di
confidenza al livello di certezza del 95%:

In [6]:
xm <- 100
sx <- 15
rho <- .95
x <- 130
t.hat <- xm + rho * (x - xm)
t.hat
se.t <- sx * sqrt(rho * (1 - rho))
se.t
t.hat + c(1, -1) * qnorm(.025, 0, 1) * se.t

[1] 128.5

[1] 3.269174

[1] 122.0925 134.9075

Dato che il limite inferiore dell'intervallo di confidenza è maggiore
del cut-off, lo psicologo conclude che il punteggio vero del rispondente
è maggiore di 120. Quindi, raccomanda che il rispondente sia ammesso al
corso avanzato.

Continuando con l'esempio precedente, supponiamo che l'attendibilità
del test abbia un valore simile a quello che solitamente si ottiene
empiricamente, ovvero 0.80.

In [7]:
xm <- 100
sx <- 15
rho <- .8
x <- 130
t.hat <- xm + rho * (x - xm)
t.hat
se.t <- sx * sqrt(rho * (1 - rho))
se.t
t.hat + c(1, -1) * qnorm(.025, 0, 1) * se.t

[1] 124

[1] 6

[1] 112.2402 135.7598

In questo secondo esempio, l'intervallo di confidenza al 95% è $[112.24,
135.76]$ e contiene il valore del cut-off. Dunque, la decisione dello
psicologo è che non vi sono evidenze sufficienti che il vero valore del
rispondente sia superiore al cut-off. Si noti come la diminuzione
dell'attendibilità del test porta all'aumento delle dimensioni
dell'intervallo di confidenza.

## Riflessioni Conclusive

La teoria classica del punteggio vero si basa su un modello additivo, in cui il punteggio osservato $X$ è considerato come la somma di due componenti: il punteggio vero stabile $T$ e il punteggio di errore casuale $E$. Si suppone che i punteggi di errore all'interno di un test non siano correlati né con i punteggi veri di quel test, né con i punteggi veri o di errore di altri test. I test paralleli hanno gli stessi punteggi veri e le stesse varianze di errore. I test che sono considerati "sostanzialmente equivalenti" o $\tau$-equivalenti, differiscono solo per una costante additiva nei punteggi veri. Tuttavia, queste assunzioni possono essere violate in presenza di diverse condizioni che influenzano i punteggi dei test. Tuttavia, poiché non possiamo osservare direttamente $T$ ed $E$, non possiamo verificare direttamente l'adeguatezza di queste assunzioni, e possiamo solo fare delle supposizioni su quando sarebbero appropriate.

È importante tenere a mente che i punteggi veri e quelli di errore sono concetti teorici e non osservabili. Ciò che possiamo osservare sono solamente i punteggi $X$. Quando parliamo di punteggi veri, è essenziale considerare che il "punteggio vero", cioè la media dei punteggi su ripetuti test indipendenti con lo stesso test, è un'astrazione teorica. Questo punteggio potrebbe non riflettere completamente l'attributo "vero" di interesse, a meno che il test non abbia una precisione perfetta, cioè che misuri esattamente ciò che afferma di misurare.

L'approccio della teoria classica dei test (CTT) nel processo di sviluppo dei test presenta diversi vantaggi. In primo luogo, i concetti della CTT sono ampiamente diffusi e comprensibili. Inoltre, sono relativamente accessibili sia per l'apprendimento che per l'applicazione. Le statistiche descrittive dei test (come la media, la deviazione standard, l'intervallo, ecc.) e le analisi degli item (in particolare la facilità e la discriminazione degli item) possono essere calcolate facilmente. Inoltre, il modello CTT risponde a varie esigenze di misurazione, specialmente nello sviluppo di valutazioni di competenze e collocazione, utili per decisioni di ammissione, confronti tra programmi e valutazioni in vari contesti lavorativi. Infine, il modello CTT permette l'interpretazione dei punteggi degli esaminati sia al 0% che al 100% e delle stime di facilità degli item da 0.0 a 1.0, riflettendo risultati realistici. Tuttavia, queste interpretazioni non sono comuni nei modelli di teoria della risposta agli item (IRT).

Tuttavia, l'adozione della CTT presenta anche alcune limitazioni. In primo luogo, i test basati sulla CTT tendono a essere lunghi e composti da elementi omogenei. In secondo luogo, gli individui che svolgono test sviluppati con il metodo CTT potrebbero essere confrontati con item troppo facili o troppo difficili per le loro abilità. In terzo luogo, i risultati dei test CTT si applicano solo al campione considerato o a campioni molto simili. In quarto luogo, tali risultati si applicano solo alla selezione corrente di item. In quinto luogo, a causa della dipendenza dalla distribuzione normale, la CTT è adatta solo per lo sviluppo di test normativi. In sesto luogo, a causa della correlazione tra discriminazione degli item, affidabilità e alcune stime di validità, gli item e i test basati sulla CTT possono risultare sensibili alle differenze agli estremi della scala. Infine, sebbene gli errori di misurazione nei test CTT varino lungo tutto il range dei possibili punteggi (ossia, l'errore standard di misurazione è minore vicino alla media e aumenta man mano che i punteggi si discostano dalla media in entrambe le direzioni), l'errore standard di misurazione stimato nei CTT rappresenta una media su tutto questo intervallo.

## Session Info

In [2]:
sessionInfo()

R version 4.3.3 (2024-02-29)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Sonoma 14.4

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.11.0

locale:
[1] C

time zone: Europe/Rome
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] psychometric_2.4  multilevel_2.7    MASS_7.3-60.0.1   nlme_3.1-164     
 [5] ggokabeito_0.1.0  viridis_0.6.5     viridisLite_0.4.2 ggpubr_0.6.0     
 [9] ggExtra_0.10.1    bayesplot_1.11.1  gridExtra_2.3     patchwork_1.2.0  
[13] semTools_0.5-6    semPlot_1.1.6     lavaan_0.6-17     psych_2.4.1      
[17] scales_1.3.0      markdown_1.12     knitr_1.45        lubridate_1.9.3  
[21] forcats_1.0.0     stringr_1.5.1     dplyr_1.1.4       purrr_1.0.2      
[25] readr_2.1.